sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessPipelineOptions.java

```diff
@@ -89,4 +89,12 @@
   boolean isBlockOnRun();
 
   void setBlockOnRun(boolean b);
+
+  @Default.Boolean(true)
+  @Description(
+      "Controls whether the runner should ensure that all of the elements of every "
+          + "PCollection are not mutated after being output.")
```

Comment:
Sentence reads a bit oddly. There's grammar, but then there's also the fact that you only talk about output, rather than input. Even though technically everything that is input was output too (derp) it is still helpful to describe in terms of properties of the `DoFn`, like "checks that a `DoFn` does not modify its inputs, nor modify its outputs after they have been output".